crates/unftp-sbe-fs/src/cap_fs.rs

```diff
@@ -0,0 +1,106 @@
+//! A capabilities-friendly workalike of tokio::fs
+// Most of these functions are copied almost verbatim from tokio::fs, but with the std parts
+// replaced by cap_std.
+
+use std::{io, path::Path, sync::Arc};
+
+use tokio::{sync::mpsc, task::spawn_blocking};
+use tokio_stream::wrappers::ReceiverStream;
+
+/// Exact copy of tokio::fs::asyncify
+async fn asyncify<F, T>(f: F) -> io::Result<T>
+where
+    F: FnOnce() -> io::Result<T> + Send + 'static,
+    T: Send + 'static,
+{
+    match spawn_blocking(f).await {
```

Comment:
Won't spawning a task for each of these slow things down?

Reply:
Yes it does.  But libunftp was already spawning a task for all of these operations.  The difference is, before the task was spawned within tokio::fs.  Now, it's spawned in libunftp-sbe-fs.  Tokio::fs relies on `spawn_blocking` for all of its operations.  It doesn't (currently) use any truly asynchronous operations, like POSIX AIO.

Reply:
Gotcha, I vaguely remember this now.